_episodes/07-checking.md

```diff
@@ -52,6 +52,7 @@ Run `make` on its own to get a full list of commands.
 
 In order to use Jekyll and/or the checking script,
 you may need to install it and some other software.
-The [setup instructions]({{ page.root }}{% link setup.md %}#optional-jekyll-setup-for-lesson-development) explain what you need and how to get it.
```

Comment:
`(Optional) Jekyll Setup for Lesson Development` title was renamed to `Jekyll Setup for Lesson Development` in 24dc11181429af418433216cf560df000ae83190 and then to `Setup for local rendering of the lessons (optional)` in  cbde2cb51b75b6d8e0765e5f3732ba96bf8da636. I propose to change `#optional-jekyll-setup-for-lesson-development` to `#setup-for-local-rendering-of-the-lessons-optional` or set the ID in the setup.md manually to something shorter.

Reply:
I attempted to fix this with c3aac9f. Let me know what you think.

Reply:
Looks good to me! Thanks!